Examples/KerrBH/KerrBHLevel.cpp

```diff
@@ -84,3 +87,26 @@ void KerrBHLevel::computeTaggingCriterion(FArrayBox &tagging_criterion,
 {
     BoxLoops::loop(ChiTaggingCriterion(m_dx), current_state, tagging_criterion);
```

Comment:
We should probably use the extraction tagging criterion to make sure we have regridded to the right level, even though we only use the low levels for this case.